Tests/ExtDataDriverGridComp.F90

```diff
@@ -603,6 +608,24 @@ subroutine run_MultipleTimes(gc, rc)
           call cap%run_one_step(status)
           _VERIFY(status)
        enddo
+    else if (cap%run_fbf) then
+       do n=1,cap%nsteps
+          call ESMF_ClockAdvance(cap%clock,rc=status)
+          _VERIFY(status)
+          call cap%run_one_step(status)
+          _VERIFY(status)
+       enddo
+       do n=1,cap%nsteps
+          call ESMF_ClockSet(cap%clock,direction=ESMF_DIRECTION_REVERSE,_RC)
+          call ESMF_ClockAdvance(cap%clock,rc=status)
+          call ESMF_ClockSet(cap%clock,direction=ESMF_DIRECTION_FORWARD,_RC)
+       enddo
```

Comment:
Can't this be done in one step?  Or are the separate substeps necessary exercise for ESMF?